modules/oneVideoBidAdapter.md

```diff
@@ -298,3 +298,89 @@ var adUnits = [
       }
   ]
 ```
+# Content Object Support
+The oneVideoBidAdapter supports passing of OpenRTB V2.5 Content Object.
+
+```
+const adUnits = [{
+            code: 'video1',
+            mediaTypes: {
+                video: {
+                    context: 'outstream',
+                    playerSize: [640, 480]
+                }
+            },
+            bids: [{
+                bidder: 'oneVideo',
+                params: {
+                    video: {
+                        playerWidth: 640,
+                        playerHeight: 480,
+                        mimes: ['video/mp4', 'application/javascript'],
+                        protocols: [2, 5],
+                        api: [1, 2],
+                        ttl: 300,
+                        content: {
+                            id: "1234",
+                            title: "Title",
+                            series: "Series",
+                            season: "Season",
+                            episode: "Episode",
```

Comment:
As per adapter `episode` is not in string as it is in `contentNumberkeys`

Reply:
fixed